src/js/auth/ManagementClient.ts

```diff
@@ -137,34 +75,32 @@ export const getUserRoles = async (userId: string): Promise<Role[]> => {
  * @param roles Array of role names (Ex: 'editor'), not Auth0 role IDs (Ex: 'rol_ds239fjdsfsd')
  */
 export const setUserRoles = async (userId: string, roles: string[]): Promise<void> => {
-  const allRoles = await auth0ManagementClient.getRoles()
-
-  const roleIdsToRemove = allRoles.reduce<string[]>((res, roleObj) => {
+  const allRoles = await auth0ManagementClient.users.getRoles()
+  const roleIdsToRemove = allRoles.data.reduce<string[]>((res, roleObj) => {
     if (roleObj.name != null && roleObj.id != null && !roles.includes(roleObj.name)) {
       res.push(roleObj.id)
     }
     return res
   }, [])
   // Removes roles that the user doesn't even have, but that's ok.
-  if (roleIdsToRemove.length > 0) await auth0ManagementClient.removeRolesFromUser({ id: userId }, { roles: roleIdsToRemove })
+  if (roleIdsToRemove.length > 0) await auth0ManagementClient.users.deleteRoles({ id: userId }, { roles: roleIdsToRemove })
```

Comment:
~~is the intent of this to delete roles from the user or delete the role from auth0 all together? it seems the original intent was to just remove a role from a user. (not familiar with this so i can be off in my assumptions)~~

actually since you're specifying users you're probably deleting from the user itself, not the entire role